TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/UsbDeviceConnection.java

```diff
@@ -0,0 +1,117 @@
+package com.novoda.tpbot.bot;
+
+import android.content.Context;
+import android.hardware.usb.UsbDevice;
+import android.hardware.usb.UsbManager;
+
+import com.novoda.notils.logger.simple.Log;
+import com.novoda.support.Optional;
+
+class UsbDeviceConnection extends DeviceConnection implements UsbChangesListener {
+
+    private final UsbChangesRegister usbChangesRegister;
```

Comment:
I was debating whether to create an interface for the `UsbChangesRegister` but tbh it is hidden in a very `android-centric` class 🤷‍♂️